tests/Routing/RoutingUrlGeneratorTest.php

```diff
@@ -128,7 +128,7 @@ public function testControllerRoutesWithADefaultNamespace()
         $route = new Illuminate\Routing\Route(['GET'], 'foo/bar', ['controller' => 'namespace\foo@bar']);
         $routes->add($route);
 
-        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => 'something\foo@bar']);
+        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => '\something\foo@bar']);
```

Comment:
@rigor789 Still, you can just simply change a working testcase to fit your need, as my argument stand, this is to test registering any route (other than in the root namespace) doesn't required `\` prefixed, and it is to that that you need to include `\` in `action()` other than the root namespace. 

That what @taylorotwell wanted/intended to ensure happen in the code.